x-pack/plugins/security_solution/public/common/components/draggables/index.test.tsx

```diff
@@ -326,5 +327,21 @@ describe('draggables', () => {
 
       expect(wrapper.find('[data-test-subj="some-field-tooltip"]').first().exists()).toBe(false);
     });
+
+    test('it uses the specified tooltipPosition', () => {
+      const wrapper = mount(
+        <TestProviders>
+          <DraggableBadge
+            contextId="context-id"
+            eventId="event-id"
+            field="some-field"
+            value="some value"
+            tooltipPosition="top"
+          />
+        </TestProviders>
+      );
+
+      expect(wrapper.find(EuiToolTip).first().props().position).toEqual('top');
```

Comment:
Tbh, I'm not proud of using `find(Component)` here since it's an implementation detail. I wasn't sure how else to test it though.

Reply:
I was thinking about adding a functional test for the suricata signature because it would implicitly test this behaviour as well. In the end I decided against it because I wasn't sure if the added test time for another functional test would be worth it. 

I added a functional test in https://github.com/elastic/kibana/pull/117874/commits/112c7a761aa9e0ce7ff2244266a7da58ef99acbc now though. Would you prefer I removed this unit test or would it be okay to keep it?

Reply:
great, thanks! And it's fine to keep this